storage/mounts.go

```diff
@@ -161,6 +161,10 @@ func getCommonClusterObject(clustersAPI clusters.ClustersAPI, clusterName string
 func getOrCreateMountingCluster(clustersAPI clusters.ClustersAPI) (string, error) {
 	cluster, err := clustersAPI.GetOrCreateRunningCluster("terraform-mount", getCommonClusterObject(clustersAPI, "terraform-mount"))
 	if err != nil {
+		// Do not treat missing cluster like a missing resource.
```

Comment:
Needed because `common.Resource` has a blanket catch-all for missing responses from APIs, but in this case, we need to prevent that from being triggered.